modules/distribution-zones/src/main/java/org/apache/ignite/internal/distributionzones/rebalance/ZoneRebalanceUtil.java

```diff
@@ -608,6 +608,24 @@ public static List<Assignments> zoneAssignmentsGetLocally(
                 .collect(toList());
     }
 
+    /**
+     * Returns partition assignments from meta storage.
+     *
+     * @param metaStorageManager Meta storage manager.
+     * @param zoneId Table ID.
+     * @param partitionId Partition ID.
+     * @return Future with partition assignments as a value.
+     */
+    public static CompletableFuture<Set<Assignment>> stablePartitionAssignments(
```

Comment:
There's a `zonePartitionAssignmentsGetLocally` method which does a similar thing. Let's call this method `zonePartitionAssignments`

Reply:
It makes sense to me. I will fix it.